apps/consent/README.md

```diff
@@ -0,0 +1,24 @@
+# login-consent-provider
```

Comment:
need better README to explain what consent is about.

`Consent`/Login is built on top of hydra to allow user to login and delegate access to their account.